pkg/mounter/proxy/server/ossfs/driver.go

```diff
@@ -29,18 +29,27 @@ func init() {
 
 // Driver manages ossfs mounts and their monitoring
 type Driver struct {
+	mounter.Mounter
 	pids           *sync.Map
 	monitorManager *server.MountMonitorManager
 	wg             sync.WaitGroup
-	raw            mount.Interface
 }
 
 func NewDriver() *Driver {
-	return &Driver{
+	driver := &Driver{
 		pids:           new(sync.Map),
 		monitorManager: server.NewMountMonitorManager(),
-		raw:            mount.NewWithoutSystemd(""),
 	}
+	m := &extendedMounter{
```

Comment:
it seems that `extendedMounter` is unnecessary.
```
	driver := &Driver{
		Mounter: mounter.NewForMounter(
			mount.NewWithoutSystemd(""),
			interceptors.OssfsSecretInterceptor,
			interceptors.OssfsMonitorInterceptor,
		),
		pids:           new(sync.Map),
		monitorManager: server.NewMountMonitorManager(),
	}
	return driver
```
And use `return h.Mounter.ExtendedMount(ctx, &mounter.MountOperation` in `Mount` instead?